### PR TITLE
[Testing] Core/ransition 3185 explicit test

### PR DESCRIPTION
**Pull request by roigcarlo**
@loumalouomega  I am gonna tear down the code so I created another branch to avoid polluting the main branch

### CODE REVIEW

**Comment by loumalouomega** [comment]
👌

**Comment by roigcarlo** [comment]
Ok without the registry works, but the tests fails obiously, lets try to activate them one by one.

File: kratos/solving_strategies/builder_and_solvers/explicit_builder.h [hunk 1]
```diff
@@ -902,6 +902,12 @@ class ExplicitBuilder
 ///@name Type Definitions
 ///@{
 
+typedef TUblasSparseSpace<double> SparseSpaceType;
```
Comment (loumalouomega) [inline kratos/solving_strategies/builder_and_solvers/explicit_builder.h]: Any guess?

Comment (loumalouomega) [reply kratos/solving_strategies/builder_and_solvers/explicit_builder.h]: @roigcarlo

Comment (roigcarlo) [reply kratos/solving_strategies/builder_and_solvers/explicit_builder.h]: to be honest I am a little bit lost, I am orbiting around the idea that pybinds tries to call the static mrComponents of the Kratoscomponents<ExplicitBuild> before that has been instantiated, the pybind bidding cannot find it, and everything crashes. But still have to do more iterations over this.

Expect it to keep failing :/

Comment (roigcarlo) [reply kratos/solving_strategies/builder_and_solvers/explicit_builder.h]: Ok nevermind, you are registering in register_factory.h

**Comment by roigcarlo** [comment]
This was already merged in #3185 and #7426

**Pull request closed without merging**